frontend/src/App.tsx

```diff
@@ -45,6 +72,36 @@ function App() {
     setNewTodoDescription("");
   };
 
+  function deleteTodo(url: string, id: string) {
+    return fetch(url + id, {
+      method: "DELETE",
+      headers: {
+        "Content-Type": "application/json",
+      },
+    });
+  }
+
+  const onDeleteTodo = async (id: string) => {
+    const response = await deleteTodo(urlString, id);
+    const { status } = await response.json();
+    if (status) {
+      setTodos((currentTodo) => currentTodo.filter((todo) => todo.id !== id));
+    }
+  };
+
+  const onChange = (name: string, value: string) => {
+    setNewTodoDescription(value);
+  };
+
+  const content = loading ? (
+    <h3 className="msg-text">LOADING...</h3>
+  ) : !loading && todos.length !== 0 ? (
+    <TodoList todos={todos} deleteTodo={onDeleteTodo}></TodoList>
+  ) : (
+    !loading &&
+    todos.length === 0 && <h3 className="msg-text">NO TASKS ADDED</h3>
+  );
+
   return (
     <div className="todo-app-wrapper">
       <div className=" card">
```

Comment:
You should make a change here. Any `input` field should ideally be inside a `form` element. This is semantically more appropriate.

A `form` also needs a `submit` button. You already have one here, and you can tweak that to submit this form.

If done correctly, you can submit the form (that is, "click" the Add button) by hitting the Enter key. In other words, you can create a Todo by just entering the description and hitting the Enter key.

**Note: Submitting the form should not refresh the page.**